tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/cassandra/CassandraTableManager.java

```diff
@@ -78,6 +80,9 @@ public CassandraTableManager(
         checkArgument(tableName.getSchema().isPresent(), "Cassandra requires table with schema");
```

Comment:
Well, the default schema was meant for removing stale tables. But we can use it for the same purpose.